setup.py

```diff
@@ -52,7 +52,7 @@
             "isort>=5.0",
             "pytest>=4.1",
             "pyro-api>=0.1.1",
-            "scipy>=1.1",
+            "scipy>=1.6",
```

Comment:
I think you can change this to `scipy>=1.6<1.7`. Newer versions seem to have bug (see https://github.com/scipy/scipy/issues/14953).